src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -317,6 +317,27 @@ public void menuItemTabulateClicked() {
     }
   }
 
+  /**
+   * Convert CVRs in current config to CDF. - Require user to save if there are unsaved changes. -
+   * Create and launch ConvertToCdfService from the saved config path.
+   */
+  public void menuItemConvertToCdfClicked() {
+    if (checkForSaveAndExecute()) {
+      if (GuiContext.getInstance().getConfig() != null) {
+        setGuiIsBusy(true);
+        ConvertToCdfService service = new ConvertToCdfService(selectedFile.getAbsolutePath());
+        service.setOnSucceeded(event -> setGuiIsBusy(false));
+        service.setOnCancelled(event -> setGuiIsBusy(false));
+        service.setOnFailed(event -> setGuiIsBusy(false));
```

Comment:
Can we factor this out into a helper? It looks like we're doing it in a few places.

Reply:
Done.